src/sentry/api/endpoints/organization_stats_v2.py

```diff
@@ -20,7 +25,12 @@ def get(self, request, organization):
                     organization,
                 )
             with sentry_sdk.start_span(op="outcomes.endpoint", description="run_outcomes_query"):
-                result_totals, result_timeseries = run_outcomes_query(query)
+                if "project_id" in query.query_groupby:
```

Comment:
nit: the structure of this could be improved slightly: since result_totals is the same in either block, you don't need it within the block. that means you can turn result_timeseries into a ternary statement.